libs/langchain-ollama/langchain.config.js

```diff
@@ -0,0 +1,22 @@
+import { resolve, dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+
+/**
+ * @param {string} relativePath
+ * @returns {string}
+ */
+function abs(relativePath) {
+  return resolve(dirname(fileURLToPath(import.meta.url)), relativePath);
+}
+
+export const config = {
+  internals: [/node\:/, /@langchain\/core\//, 'ollama/browser'],
```

Comment:
cc @jacoblee93 I think I did this correctly? (`'ollama/browser'` was giving me rollup errors)

Reply:
Yeah I think so